runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -289,4 +289,21 @@ static String htmlLabel(String key, int value)
 		String valueStr = StackFormatter.quantityToRSDecimalStack(value, true);
 		return String.format(HTML_LABEL_TEMPLATE, ColorUtil.toHexColor(ColorScheme.LIGHT_GRAY_COLOR), key, valueStr);
 	}
+
+	void toggleCanvasItemText()
+	{
+		if (canvasItem.getText().equals(REMOVE_STATE))
+		{
+			canvasItem.setText(ADD_STATE);
+		}
+		else
+		{
+			canvasItem.setText(REMOVE_STATE);
+		}
+	}
+
+	boolean getPaused()
```

Comment:
Use a `@Getter` annotation on the field instead. (and use an `AccessLevel` if it should be package-level getter only)

Reply:
are the new changes good?